appium/tests/helpers/ElementHelper.ts

```diff
@@ -60,6 +60,29 @@ class ElementHelper {
     await this.waitAndClick(element);
     await element.setValue(text);
   }
+
+  //wait for text in element during 15 seconds (if the text doesn't appear during 15s, it will show the error)
+  static waitForText = async (element: WebdriverIO.Element, text: string, timeout: number = DEFAULT_TIMEOUT) => {
+    await this.waitElementVisible(element);
+    await element.waitUntil(async  function () {
+      return (await element.getText() === text)
+    }, {
+      timeout: timeout,
+      timeoutMsg: 'expected text to be different after 5s'
+    });
+  }
+
+  //wait for value in element during 15 seconds (if the value doesn't appear during 15s, it will show the error)
+  static waitForValue = async (element: WebdriverIO.Element, value: string, timeout: number = DEFAULT_TIMEOUT) => {
+    await this.waitElementVisible(element);
+    await element.waitUntil(async  function () {
+      return (await element.getValue() === value)
+    }, {
+      timeout: timeout,
+      timeoutMsg: 'expected value to be different after 15s'
```

Comment:
Do `expected value within ${timeout}s to be "${value}" but got last value "${lastValue}"`

Similarly above